include/bluetooth/mesh/main.h

```diff
@@ -425,14 +425,107 @@ int bt_mesh_lpn_set(bool enable);
  */
 int bt_mesh_lpn_poll(void);
 
-/** @brief Register a callback for Friendship changes.
+/** Low Power Node callback functions. */
+struct bt_mesh_lpn_cb {
+	/** @brief Friendship established.
+	 *
+	 *  This callback notifies the application that friendship has
+	 *  been successfully established.
+	 *
+	 *  @param net_idx  NetKeyIndex used during established friendship.
+	 *  @param friend_addr Friend address.
+	 *  @param queue_size  Friend queue size.
+	 *  @param recv_window Low Power Node's listens duration for
+	 *  Friend response.
+	 */
+	void (*established)(uint16_t net_idx, uint16_t friend_addr,
+			    uint8_t queue_size, uint8_t recv_window);
+
+	/** @brief Friendship terminated.
+	 *
+	 *  This callback notifies the application that friendship has
+	 *  been terminated.
+	 *
+	 *  @param net_idx  NetKeyIndex used during established friendship.
+	 *  @param friend_addr Friend address.
+	 */
+	void (*terminated)(uint16_t net_idx, uint16_t friend_addr);
+
+	/** @brief Local Poll Request.
+	 *
+	 *  This callback notifies the application that the local node has
+	 *  polled the friend node.
+	 *
+	 *  This callback will be called before @ref bt_mesh_lpn_cb::established
+	 *  when attempting to establish a friendship.
+	 *
+	 *  @param net_idx  NetKeyIndex used during established friendship.
```

Comment:
```suggestion
	 *  @param net_idx  NetKeyIndex used during friendship establishment.
```

Same for the other callbacks